pkg/cephfs/driver.go

```diff
@@ -108,9 +108,9 @@ func (fs *Driver) Run(conf *util.Config, cachePersister util.CachePersister) {
 		}
 	} else {
 		// Pick the first available mounter as the default one.
```

Comment:
Can this be simplifed as, 'pick kernel mounter and kernel version >= 4.17 if available, else pick fuse mounter'?

Reply:
Done